Refactor guides section + promote HTTP over GIT (#1338)
huggingface/huggingface_hub: docs/source/installation.mdx

- `huggingface_hub`'s cache system relies on symlinks to efficiently cache files downloaded
from the Hub. On Windows, you must activate developer mode or run your script as admin to
enable symlinks. If they are not activated, the cache-system still works but in an non-optimized

> manner. Please read [this page](./guides/manage-cache#limitations) for more details.

Reviewer: ```suggestion
manner. Please read [the cache limitations](./guides/manage-cache#limitations) section for more details.
```
